decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -7,21 +7,28 @@
 <%= render partial: "voting_rules" %>
 <div class="row column view-header">
   <h2 class="heading2"><%= @proposal.title %></h2>
-  <%= cell "decidim/author_box", present(@proposal).author, extra: (capture do %>
+  <!-- AUTHOR-CEL!!! -->
+  <%= card_for present(@proposal).author, from: @proposal %>
+
+  <% if false %>
+    <%= cell "decidim/author_box", present(@proposal).author, extra: (capture do %>
       <span><%= l @proposal.created_at, format: :decidim_short %></span>
       <button type="button" data-open="<%= current_user.present? ? "flagModal" : "loginModal" %>" title="<%= t(".report") %>" aria-controls="<%= current_user.present? ? "flagModal" : "loginModal" %>" aria-haspopup="true" tabindex="0">
         <%= icon "flag", aria_label: t(".report"), class: "icon--small" %>
       </button>
       <% unless @proposal.official? %>
+        <p>official</p>
```

Comment:
Remove this (also, you're checking if the proposal is *not* official)

Reply:
this is going to be removed also